src/v/raft/heartbeat_manager.cc

```diff
@@ -205,11 +205,6 @@ void heartbeat_manager::process_reply(
             }
             // propagate error
             (*it)->get_probe().heartbeat_request_error();
-            (*it)->process_append_entries_reply(
```

Comment:
it's not clear why this is being deleted. presumably this patch is only replacing node_id with vnode...